projects/fluxcd/flux2/docker/linux/Dockerfile

```diff
@@ -0,0 +1,26 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+ARG BASE_IMAGE
+FROM $BASE_IMAGE
+
+ARG TARGETARCH
+ARG TARGETOS
+
+COPY _output/dependencies/linux-$TARGETARCH/eksd/kubernetes/client/bin/kubectl /usr/local/bin/kubectl
```

Comment:
nit: just for consistency, and im sure weve missed this elsewhere, but instead of linux, $TARGETOS

Reply:
Done